src/Evolution/Systems/Cce/Actions/InitializeWorldtubeBoundary.hpp

```diff
@@ -0,0 +1,99 @@
+// Distributed under the MIT License.
```

Comment:
Can you factor the changes in this commit into several smaller commits? There appear to be a lot of fairly independent changes.